scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,475 @@
+# -*- coding: utf-8 -*-
+"""
+This script computes four myelin indices maps from the Magnetization Transfer
+(MT) and inhomogeneous Magnetization Transfer (ihMT) images. Magnetization
+Transfer is a contrast mechanism in tissue resulting from the proton exchange
+between non-aqueous protons (from macromolecules and their closely associated
+water molecules, the "bound" pool) and protons in the free water pool called
+aqueous protons. This exchange attenuates the MRI signal, introducing
+microstructure-dependent contrast. MT's effect reflects the relative density
+of macromolecules such as proteins and lipids, it has been associated with
+myelin content in white matter of the brain.
+
+Different contrasts can be done with an off-resonance pulse prior to image
+acquisition (a prepulse), saturating the protons on non-aqueous molecules,
+by applying different frequency irradiation. The two MT maps and two ihMT maps
+are obtained using five contrasts: single frequency positive or negative and
+dual frequency with an alternation of both positive and negative frequency
+(saturated images); and one unsaturated contrast as reference (T1weighted).
+
+
+The Input Data :
+  - it is recommended to use dcm2niix (v1.0.20200331) to convert data
+    https://github.com/rordenlab/dcm2niix/releases/tag/v1.0.20200331
+  - all contrasts must have a same number of echoes
+  - all input must have a matching json with the same filename
+
+The output consist in two types of images:
+        five contrast images and four (ih)MT maps.
```

Comment:
Can you enumerate the output MT files a little better? 
For instance, MTsat.nii.gz, MTR.nii.gz, ihMTsat.nii.gz, ihMTR.nii.gz are the outputs